composer.json

```diff
@@ -11,18 +11,20 @@
   ],
   "require": {
     "php": ">=7.1",
+    "ext-json": "*",
     "ext-pdo": "*",
     "keboola/csv": "^1.1",
     "keboola/db-extractor-logger": "^1.0",
     "keboola/db-extractor-ssh-tunnel": "^1.1",
     "keboola/php-datatypes": "^4.0",
-    "keboola/php-utils": "^2.3",
+    "keboola/php-temp": "^2.0",
```

Comment:
Prošel jsi všechna použití? Ve v2 se změnilo mazání (už není v automatické, ale musí se udělat ručně).

Reply:
jj prošel - používá se to tady jen na jednom místě viz tvůj comment:
https://github.com/keboola/db-extractor-common/pull/127#discussion_r319965739